Moolya.robot

```diff
@@ -0,0 +1,152 @@
+*** Settings ***
+Library          SeleniumLibrary
+Resource         C:\\Users\\Admin\\Documents\\GitHub\\Mentorship_Abhijit\\BrowserKeywords.robot
+Suite Setup      Open Initial Browser
+Suite Teardown   Close All Browsers
+
+# All Keywords and variables are imported from BrowserKeywords robot file
+
+# To run Press Cntrl+Shift+P  in Windows
+# To run Press cmd-shift-P  in Mac
+
+
+
+*** Test Cases ***
+
+#To Setup Window
+Browser Setup
+    Window Maximize 
+    Selenium wait 
+
+
+#To Verify Moolya Website is Opened or Not
+Verify Moolya Website is Opened
+
```

Comment:
Remove newline in between the test titles and keywords invoking lines

Reply:
Ok, will do that